Submitting XCActionBar plugin package spec

pdcgomes opened the pull request:
Requesting for XCActionBar publishing.

jurre commented:
Hey, I don't think this will work because the name listed in the package is `XCActionBar` but the name of the xcodeproj is `PGXcodeActionBrowser`. From the readme:

> The url should be a repository containing the Xcode project, and the name should be the same as the .xcodeproj file.

pdcgomes commented:
Thanks! 
Yeah, I started without really knowing what it would be called and changed my mind in the process. I just renamed the project file so it matches the expectation.

jurre commented:
Thanks!
